pkg/volume/util.go

```diff
@@ -148,3 +148,20 @@ func CalculateTimeoutForVolume(minimumTimeout, timeoutIncrement int, pv *api.Per
 func RoundUpSize(volumeSizeBytes int64, allocationUnitBytes int64) int64 {
 	return (volumeSizeBytes + allocationUnitBytes - 1) / allocationUnitBytes
 }
+
+// GenerateVolumeName returns a PV name with a random suffix.
```

Comment:
Comment is not quite accurate. Not a `random suffix`. Suffix is passed in as a parameter.